test/e2e/tests/from-import-ui.spec.js

```diff
@@ -218,7 +219,8 @@ describe('MetaMask Import UI', function () {
         // choose Create account from the account menu
         await driver.clickElement({ text: 'Import account', tag: 'div' });
         // enter private key
-        await driver.fill('#private-key-box', testPrivateKey2);
+        await driver.delay(regularDelayMs);
+        await driver.fill('#private-key-box', testPrivateKey2, 2);
```

Comment:
One question for my understanding here. Why do we need this third argument for delay? What does it do?

Reply:
So if we don't mock a delay, the input will be wiped out. I guess it is something to do with event bubbling, but if we wait until input has taken the field, and not switch to button immediately. this could solve the flakyness